csharp/ql/lib/semmle/code/csharp/commons/QualifiedName.qll

```diff
@@ -0,0 +1,34 @@
+/**
+ * Provides predicates related to C# qualified name.
+ */
+
+/**
+ * Returns the concatenation of `qualifier` and `name`, separated by a dot.
+ */
+bindingset[namespace, name]
+string printQualifiedName(string namespace, string name) {
```

Comment:
Yes, that is a good idea (I also considered that, but thought it might be confusing since we are talking about getting rid of getQualifiedName). In any case, if you also think we should use `getQualifiedName` then we go for that.